crates/cli/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+extern crate swc_malloc;
+
```

Comment:
This isn't necessary.
```suggestion
```

Reply:
This is necessary.
https://github.com/rust-lang/rust/issues/64402

Reply:
Probably best to communicate that requirement to any future maintainers that may likewise see it as redundant:

```suggestion
// Required: https://github.com/rust-lang/rust/issues/64402#issuecomment-530852886
extern crate swc_malloc;

```